src/objects/components/HistoryYearDropdown.js

```diff
@@ -8,20 +8,22 @@ export default function HistoryYearDropdown() {
   const { noradNumber, yearLaunched } = useObjectsState();
   const [yearChosen, setYearChosen] = useState(null);
   const [{ isLoading, isError, data }, doFetch] = useTrusatGetApi();
+  // Used to render the rows of years from present year to the year of launch
+  const currentYear = new Date().getFullYear();
 
   useEffect(() => {
     if (noradNumber && yearChosen) {
       doFetch(`/object/history?year=${yearChosen}&norad_number=${noradNumber}`);
     }
     if (yearChosen === null) {
-      setYearChosen(2020);
+      setYearChosen(currentYear);
     }
-  }, [noradNumber, yearLaunched, yearChosen, doFetch, data]);
+  }, [noradNumber, yearLaunched, yearChosen, doFetch, data, currentYear]);
```

Comment:
@mainnebula Nice. 

I added `currentYear` to the "dependency array" - You can learn more about why this is important when you dig more into how `React hooks` work.